.gitignore

```diff
@@ -2,10 +2,15 @@
 output-site
 output
 _site/
+build/
+workflow/
+vendor/
 .sconsign.dblite
 html/index.html
 pdf
 *.pyc
+*.nbconvert.ipynb
+*.html
```

Comment:
Is this going to cause any problems for the `gh-pages` branch, or are the generated pages force-added?

Reply:
Right! maybe the `html` shouldn't be there!